activemodel/lib/active_model/type/value.rb

```diff
@@ -19,7 +25,8 @@ def serializable?(value)
         true
       end
 
-      def type # :nodoc:
+      # Returns the unique type name as a Symbol.
```

Comment:
I was thinking we should add a note like following, since this method returns `nil` by default (not a Symbol):

```suggestion
      # Returns the unique type name as a Symbol. Subclasses should override
      # this method.
```

But `Value` instances clearly work with this method returning `nil`, so returning a Symbol is not strictly necessary.  Does anyone know what could happen if a subclass does not override this method?

I suppose it's good practice to override the method anyway, so adding the note wouldn't hurt.